llvm/include/llvm/CodeGen/MIRPrinter.h

```diff
@@ -14,14 +14,33 @@
 #ifndef LLVM_CODEGEN_MIRPRINTER_H
 #define LLVM_CODEGEN_MIRPRINTER_H
 
+#include "llvm/CodeGen/MachinePassManager.h"
+#include "llvm/Support/Debug.h"
+
 namespace llvm {
 
 class MachineBasicBlock;
 class MachineFunction;
 class Module;
-class raw_ostream;
 template <typename T> class SmallVectorImpl;
 
+class PrintMIRPreparePass : public MachinePassInfoMixin<PrintMIRPreparePass> {
+  raw_ostream &OS;
+
+public:
+  PrintMIRPreparePass(raw_ostream &OS = dbgs()) : OS(OS) {}
```

Comment:
actually `errs()` makes more sense, it's less debugging and more something you're actually expecting to be output